src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Core/Models/ResourceRegistry/ConsentMetadata.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Altinn.AccessManagement.Core.Models.ResourceRegistry
+{
+    /// <summary>
+    /// Model describing the consent metadata for a resource
+    /// </summary>
+    public class ConsentMetadata
+    {
+        /// <summary>
+        /// Define if metadata is optional
+        /// </summary>
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
```

Comment:
Why `JsonIgnoreCondition.WhenWritingDefault`? This just makes it harder to deal with and understand the JSON.

Reply:
Currently it is not supported so keeping it as is. I am not even sure we ever will support it.